common/stacks/src/main/scala/weco/api/stacks/models/HoldResponse.scala

```diff
@@ -17,6 +17,7 @@ object HoldRejected {
   case object ItemCannotBeRequested extends HoldRejected
   case object ItemIsOnHoldForAnotherUser extends HoldRejected
   case object ItemMissingFromSourceSystem extends HoldRejected
+  case object ItemUnavailableFromSourceSystem extends HoldRejected
```

Comment:
What's the difference between "missing" and "unavailable"?

Reply:
I've renamed this status to `SourceSystemNotSupported`, to denote a rejected attempt to place a hold on a source system we can't create holds for.